cogs/quote_commands.py

```diff
@@ -146,7 +147,7 @@ async def get(self, ctx:utils.Context, identifier:commands.clean_content):
                 embed.set_image(data['text'])
             else:
                 embed.description = data['text']
-            embed.set_footer(text=f"Quote ID {data['quote_id'].upper()}")
+            embed.set_footer(text=f"Quote ID ({data['quote_id'].upper()})[{}]")
```

Comment:
You're missing a link here. You're also missing a check for quotes that _don't have_ message links.